image/distributionutil/registry.go

```diff
@@ -29,6 +29,7 @@ func NewV2Repository(named reference.Named, actions ...string) (distribution.Rep
 	authConfig, err := utils.GetDockerAuthInfoFromDocker(named.Domain())
 	if err != nil {
 		logger.Warn("failed to get auth info, err: %s", err)
```

Comment:
This is not the point.
sometimes we do not need to login to registry(like sealer pull).